dapp/src/app/components/OpenColony/index.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+
+type Props = {
+  count: number;
+  setAddress: (address: string) => any;
+  getCount: () => any;
+};
+
+export class OpenColony extends React.Component<Props, any> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
```

Comment:
NIT: Can we create a State type above and use it in the template "React.Component<Props, State>"?

Here's an example (that uses interfaces instead of types):
https://github.com/JingyuZ/GithubColony/blob/sogasg/integrate-colonyjs/dapp/src/app/components/TodoItem/index.tsx

Reply:
I'd like to follow the design pattern followed in the TodoItem\index.tsx linked above, but this is all pedantic at this point.

Also a lot of this "getting started" code will most likely be rewritten for better object oriented design, so we can fix this up then.